src/app/learn/overview/index.html

```diff
@@ -35,7 +35,7 @@
   <ul>
     <li>
       <p>
-        SKY UX takes advantage of <a href="https://angular.io/">Angular</a> to keep SKY UX modern while increasing the tooling, testing, and performance available. Components, services, and modules are open source in <a href="https://github.com/blackbaud/skyux2">the SKY UX GitHub repo</a>.
+        SKY UX takes advantage of <a href="https://angular.io/">Angular</a> to keep SKY UX modern while increasing the tooling, testing, and performance available. Components, services, and modules are open source in multiple GitHub repositories that target different areas of development.
```

Comment:
This change looks good, but it just occurred to me that maybe we should link from the new text to the packages list at https://developer.blackbaud.com/skyux/contribute/packages. What do you think?

Reply:
Added link